dmcontent/converters.py

```diff
@@ -0,0 +1,46 @@
+from flask._compat import string_types
```

Comment:
We could change this to `six.string_types`, and then we don't need Flask as a dependency (need `six` though).